google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -2163,7 +2163,7 @@ def create_notification topic, custom_attrs: nil, event_types: nil,
                       prefix: prefix, payload: payload,
                       user_project: user_project }
 
-          gapi = service.insert_notification name, topic, options
+          gapi = service.insert_notification name, topic, **options
```

Comment:
I'd prefer to pass keyword arguments directly here, since there is no `delete_if` on the options hash:

```rb
          gapi = service.insert_notification name, topic, custom_attrs: custom_attrs,
                                                          event_types: event_types,
                                                          prefix: prefix,
                                                          payload: payload,
                                                          user_project: user_project
```